### ISSUE TITLE
Support parallel repair

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#211]
This issue is a subset of #79

"Unlike sequential repair, parallel repair constructs the Merkle tables for all nodes at the same time. Therefore, no snapshots are required (or generated). Use a parallel repair to complete the repair quickly or when you have operational downtime that allows the resources to be completely consumed during the repair."
http://docs.datastax.com/en/cassandra/2.1/cassandra/tools/toolsRepair.html

In other words: parallel repair is used when the system is down
